src/types/jobDescription.ts

```diff
@@ -0,0 +1,356 @@
+                <vscode-data-grid-cell grid-column="2">${this.jobDescriptionInfo?.logClProgramCommands}</vscode-data-grid-cell>
+              </vscode-data-grid-row>
+              <vscode-data-grid-row>
+                <vscode-data-grid-cell grid-column="1">Device Recovery Action</vscode-data-grid-cell>
+                <vscode-data-grid-cell grid-column="2">${this.jobDescriptionInfo?.deviceRecoveryAction}</vscode-data-grid-cell>
+              </vscode-data-grid-row>
+              <vscode-data-grid-row>
+                <vscode-data-grid-cell grid-column="1">Time Slice End Pool</vscode-data-grid-cell>
+                <vscode-data-grid-cell grid-column="2">${this.jobDescriptionInfo?.timeSliceEndPool}</vscode-data-grid-cell>
+              </vscode-data-grid-row>
+              <vscode-data-grid-row>
+                <vscode-data-grid-cell grid-column="1">Allow Multiple Threads</vscode-data-grid-cell>
+                <vscode-data-grid-cell grid-column="2">${this.jobDescriptionInfo?.allowMultipleThreads}</vscode-data-grid-cell>
+              </vscode-data-grid-row>
+              <vscode-data-grid-row>
+                <vscode-data-grid-cell grid-column="1">Workload Group</vscode-data-grid-cell>
+                <vscode-data-grid-cell grid-column="2">${this.jobDescriptionInfo?.workloadGroup}</vscode-data-grid-cell>
+              </vscode-data-grid-row>
+              <vscode-data-grid-row>
+                <vscode-data-grid-cell grid-column="1">ASPGRP</vscode-data-grid-cell>
+                <vscode-data-grid-cell grid-column="2">${this.jobDescriptionInfo?.aspgrp}</vscode-data-grid-cell>
+              </vscode-data-grid-row>
+              <vscode-data-grid-row>
+                <vscode-data-grid-cell grid-column="1">DDM Conversation</vscode-data-grid-cell>
+                <vscode-data-grid-cell grid-column="2">${this.jobDescriptionInfo?.ddmConversation}</vscode-data-grid-cell>
+              </vscode-data-grid-row>
+            </vscode-data-grid>
+            `;
+
+        const libraryListTab = `<h1>Library list</h1>
+            <vscode-data-grid>
+              <vscode-data-grid-row row-type="header">
+                <vscode-data-grid-cell cell-type="columnheader" grid-column="1"><b>Sequence</b></vscode-data-grid-cell>
+                <vscode-data-grid-cell cell-type="columnheader" grid-column="2"><b>Library</b></vscode-data-grid-cell>
+              </vscode-data-grid-row>
+              ${this.jobDescriptionLibraries?.map(library => {
+                  return /*html*/`
+                  <vscode-data-grid-row>
+                    <vscode-data-grid-cell grid-column="1">${library.sequence}</vscode-data-grid-cell>
+                    <vscode-data-grid-cell grid-column="2">${library.library}</vscode-data-grid-cell>
+                  </vscode-data-grid-row>`;
+                }).join("")}
+            </vscode-data-grid>
+            `;
```

Comment:
Try using the webview toolkit's `Components.dataGrid` here:
```suggestion
       const libraryListTab = /* html */ `<h1>Library list</h1>
          ${Components.dataGrid<JobDescriptionLibrary>({
              columns:[
                  {title: "Sequence", cellValue: l => String(l.sequence)},
                  {title: "Library", cellValue: l => l.library}
              ]
          }, jobDescriptionLibraries)}`;
```